tests/ui/traits/next-solver/normalize-self-type-constrains-trait-args.rs

```diff
@@ -0,0 +1,24 @@
+//@ check-pass
+
+// This goal is also possible w/ a GAT, but lazy_type_alias
+// makes the behavior a bit more readable.
+#![feature(lazy_type_alias)]
+//~^ WARN the feature `lazy_type_alias` is incomplete
```

Comment:
```suggestion
#![allow(incomplete_features)]
```

Reply:
I actually quite like the explicit warn since it'll cause an error (and force it to be removed) when the feature is marked as not-incomplete -- rather than having this `allow` stay around forever.